lib/new-package.js

```diff
@@ -305,6 +305,7 @@ ElmjutsuDumMyM0DuL3.elm
   const licenseArgs = options.forTests ? '--name "Test User" --year 2020' : '';
   try {
     childProcess.execSync(
+      // TODO(@lishaduck): Evaluate calling the API instead.
```

Comment:
Pros:
- Direct call (perf)
- Clearer dependency chain
- Works even if npx isn't on path

Cons:
- Would show up as unmaintained on socket, etc
- Depends on deprecated packages
- Bloats `node_modules` for 99% of users

Reply:
We could also inline the functionality of the package: support only the common ones and let the user opt-out if they want a different one.